integration/kubernetes/k8s-port-forward.bats

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+source /etc/os-release || source /usr/lib/os-release
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	get_pod_config_dir
+
+	# Check redis-cli is installed
+	check_package=$(which redis-cli | wc -l)
+
+	# Install redis-cli if is not installed
+	if [ "$check_package" -eq 0 ]; then
+		case "$ID" in
+			ubuntu | debian )
+				sudo -E apt install -y redis-server
+				;;
+			fedora )
+				sudo -E dnf -y install redis
+				;;
+			centos | rhel )
+				sudo -E yum install -y redis
+				;;
+			opensuse-* | sled | sles )
+				sudo -E zypper install -y redis
+				;;
+		esac
+	fi
+}
+
+@test "Port forwarding" {
+	deployment_name="redis-master"
+
+	# Create deployment
+	kubectl apply -f "${pod_config_dir}/redis-master-deployment.yaml"
+
+	# Check deployment
+	kubectl wait --for=condition=Available deployment/"$deployment_name"
+	kubectl expose deployment/"$deployment_name"
+
+	# Get pod name
+	pod_name=$(kubectl get pods --output=jsonpath={.items..metadata.name})
+	kubectl wait --for=condition=Ready pod "$pod_name"
+
+	# View replicaset
+	kubectl get rs
+
+	# Create service
+	kubectl apply -f "${pod_config_dir}/redis-master-service.yaml"
+
+	# Check service
+	kubectl get svc | grep redis
+
+	# Check redis service
+	port_redis=$(kubectl get pods $pod_name --template='{{(index (index .spec.containers 0).ports 0).containerPort}}{{"\n"}}')
+
+	# Verify that redis is running in the pod and listening on port
+	port=6379
+	[ "$port_redis" -eq "$port" ]
+
+	# Forward a local port to a port on the pod
+	(2&>1 kubectl port-forward "$pod_name" 7000:"$port"> /dev/null) &
```

Comment:
This will create file `1` which probably isn't what you want. Since bats uses bash, it would be clearest to say:

```bash
(kubectl port-forward "$pod_name" 7000:"$port" &>/dev/null) &
```